### PR TITLE
first commit of 1.13.0 changelog

### PR DESCRIPTION
**Pull request by allencloud**
not finished, still need following work.

Signed-off-by: allencloud <allen.sun@daocloud.io>

### CODE REVIEW

File: docker-1.13.0-changelog.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* 为Splunk的日志驱动提高性能和可靠性
+* 为Splunk的日志驱动格式化配置项，同时跳过验证连接
+
+## 网络
+
+* 为每个网络添加`--attachable`参数，使得支持容器在运行时加入swarm mode的overlay网络中
+* 在`docker service create`中使用`--publish`选项时，添加了任务单主机端口暴露的功能
+* 为Windows server 2016添加overlay网络驱动的支持（需要即将到来的ws2016更新）
+* 在IPTables中修改默认的 `FOWARD` 链为`DROP`
+* 在windows机器上，添加新的功能，使得允许用户指定特定的IP地址到预留的网络上
```
Comment (AkihiroSuda) [inline docker-1.13.0-changelog.md]: 错字: FORWARD

File: docker-1.13.0-changelog.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+* 当docker的标签被用作journald的名称时，审查这些labels标签
+* 修复了 `docker logs --tail` 显示的日志行数比实际少的错误
+* 为Splunk的日志驱动提高性能和可靠性
+* 为Splunk的日志驱动格式化配置项，同时跳过验证连接
+
+## 网络
+
+* 为每个网络添加`--attachable`参数，使得支持容器在运行时加入swarm mode的overlay网络中
+* 在`docker service create`中使用`--publish`选项时，添加了任务单主机端口暴露的功能
+* 为Windows server 2016添加overlay网络驱动的支持（需要即将到来的ws2016更新）
```
Comment (AkihiroSuda) [inline docker-1.13.0-changelog.md]: 已经来了 😀（Windows10而已） https://blogs.technet.microsoft.com/virtualization/2017/02/09/overlay-network-driver-with-support-for-docker-swarm-mode-now-available-to-windows-insiders-on-windows-10/

File: docker-1.13.0-changelog.md (hunk 1 above)
Comment (allencloud) [reply docker-1.13.0-changelog.md]: That is great, Thanks a lot. Updated now 😄

File: docker-1.13.0-changelog.md (hunk 2 above)
Comment (allencloud) [reply docker-1.13.0-changelog.md]: Thanks for the feedback and provided reference. Updated as well.
While what makes me excited is that the reason why you are so familiar with Chinese. A Chinese? or interest in Chinese? 😄

Comment (AkihiroSuda) [reply docker-1.13.0-changelog.md]: 汉语和日语，多少语词相似，所以看得懂一点点汉语 (但是完全听不懂 😭 )

Comment (allencloud) [reply docker-1.13.0-changelog.md]: That is great. Maybe I could begin to learn some Japanese 😸

**Comment by allencloud** [comment]
LGTM

**Pull request merged**